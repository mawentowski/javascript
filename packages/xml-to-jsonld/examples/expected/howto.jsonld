{
  "@context": "https://schema.org",
  "@type": "HowTo",
  "name": "How to Brew Pour-Over Coffee",
  "description": "A quick guide to consistent, tasty pour-over coffee at home.",
  "totalTime": "PT6M",
  "supply": [
    {
      "@type": "HowToSupply",
      "name": "Coffee beans",
      "requiredQuantity": "20 g"
    },
    {
      "@type": "HowToSupply",
      "name": "Water",
      "requiredQuantity": "300 ml"
    }
  ],
  "tool": [
    {
      "@type": "HowToTool",
      "name": "Gooseneck kettle"
    },
    {
      "@type": "HowToTool",
      "name": "Dripper + filter"
    }
  ],
  "step": [
    {
      "@type": "HowToStep",
      "name": "Heat water",
      "text": "Bring water to ~96°C (205°F)."
    },
    {
      "@type": "HowToStep",
      "name": "Rinse filter",
      "text": "Rinse paper filter and pre-heat the dripper."
    },
    {
      "@type": "HowToStep",
      "name": "Bloom",
      "text": "Pour ~40 ml to wet grounds, wait 30–45 s."
    }
  ]
}