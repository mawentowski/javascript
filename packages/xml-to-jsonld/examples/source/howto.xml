<!-- <?xml-model href="../packages/schemas/rnc/site-seo.rnc"
            type="application/relax-ng-compact-syntax"?> -->


<HowTo>
  <name>How to Brew Pour-Over Coffee</name>
  <description>A quick guide to consistent, tasty pour-over coffee at home.</description>
  <totalTime>PT6M</totalTime>
  <supply>
    <HowToSupply>
      <name>Coffee beans</name>
      <requiredQuantity>20 g</requiredQuantity>
    </HowToSupply>
  </supply>
  <supply>
    <HowToSupply>
      <name>Water</name>
      <requiredQuantity>300 ml</requiredQuantity>
    </HowToSupply>
  </supply>
  <tool>
    <HowToTool>
      <name>Gooseneck kettle</name>
    </HowToTool>
  </tool>
  <tool>
    <HowToTool>
      <name>Dripper + filter</name>
    </HowToTool>
  </tool>
  <step>
    <HowToStep>
      <name>Heat water</name>
      <text>Bring water to ~96°C (205°F).</text>
    </HowToStep>
  </step>
  <step>
    <HowToStep>
      <name>Rinse filter</name>
      <text>Rinse paper filter and pre-heat the dripper.</text>
    </HowToStep>
  </step>
  <step>
    <HowToStep>
      <name>Bloom</name>
      <text>Pour ~40 ml to wet grounds, wait 30–45 s.</text>
    </HowToStep>
  </step>
</HowTo>