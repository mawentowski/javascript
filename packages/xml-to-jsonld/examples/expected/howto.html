<h1>How to Brew Pour-Over Coffee</h1><p>A quick guide to consistent, tasty pour-over coffee at home.</p><h2>Supplies</h2><ul><li>Coffee beans — 20 g</li><li>Water — 300 ml</li></ul><h2>Tools</h2><ul><li>Gooseneck kettle</li><li>Dripper + filter</li></ul><h2>Steps</h2><ol><li><strong>Heat water</strong><div>Bring water to ~96°C (205°F).</div></li><li><strong>Rinse filter</strong><div>Rinse paper filter and pre-heat the dripper.</div></li><li><strong>Bloom</strong><div>Pour ~40 ml to wet grounds, wait 30–45 s.</div></li></ol>